sunpycube/cube/NDCube.py

```diff
@@ -165,7 +167,18 @@ def world_to_pixel(self, quantity_axis_list, origin=0):
         return result[::-1]
 
     def to_sunpy(self):
-        raise NotImplementedError("to_sunpy() is not Implemented.")
+        wcs_axes = list(self.wcs.wcs.ctype)
+        missing_axis = self.missing_axis
+        index_not_one = []
+        if 'TIME' in wcs_axes and len(self.dimensions.shape) is 1:
+            result = self.pixel_to_world([u.Quantity(self.data, unit=u.pix)])
+        elif 'HPLT-TAN' in wcs_axes and 'HPLN-TAN' in wcs_axes and len(self.dimensions.shape) is 2:
+            if not missing_axis[wcs_axes.index("HPLT-TAN")] and not missing_axis[wcs_axes.index("HPLN-TAN")]:
+                result = sunpy.map.Map(self.data, self.meta)
+        else:
+            warnings.warn("Object type not Implemented")
+            result = None
```

Comment:
This should raise a NotImplementedError.  The message should list the object that are implemented so the user doesn't think the function doesn't work at all.